commit 6424bf700c1851fc7e86b83959a8578a401d80c1
Author: Xose Vazquez Perez <xose.vazquez@gmail.com>
Date:   Sun Mar 28 17:48:05 2010 +0200

    wireless: rt2x00: rt2800pci: new id
    
    Taken from latest ralink linux driver, it's a RT3593 PCI/PCIe.
    
    Signed-off-by: Xose Vazquez Perez <xose.vazquez@gmail.com>
    Acked-by: Ivo van Doorn <IvDoorn@gmail.com>
    Acked-by: Gertjan van Wingerde <gwingerde@gmail.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

--- a/drivers/net/wireless/rt2x00/rt2800pci.c
+++ b/drivers/net/wireless/rt2x00/rt2800pci.c
@@ -1212,6 +1212,7 @@ static DEFINE_PCI_DEVICE_TABLE(rt2800pci
 	{ PCI_DEVICE(0x1814, 0x3062), PCI_DEVICE_DATA(&rt2800pci_ops) },
 	{ PCI_DEVICE(0x1814, 0x3562), PCI_DEVICE_DATA(&rt2800pci_ops) },
 	{ PCI_DEVICE(0x1814, 0x3592), PCI_DEVICE_DATA(&rt2800pci_ops) },
+	{ PCI_DEVICE(0x1814, 0x3593), PCI_DEVICE_DATA(&rt2800pci_ops) },
 #endif
 	{ 0, }
 };
